shelley-ma/formal-spec/value.tex

```diff
@@ -272,26 +223,11 @@ \subsection{Multi-assets and $\Value$ in the Mary Era}
 (that could have a hash-associated policy) eliminates the possibility
 certain cryptographic attacks.
 We sometimes refer to Ada as the primary or principal currency. Ada does not,
-for the purposes of the Mary ledger spec, have a $\PolicyID$ or an $\AssetName$.
+for the purposes of the Mary ledger specification, have a $\PolicyID$ or an $\AssetName$.
 
-\subsection{Pointwise Value Operations and Partial Order}
+\subsection{Fixing a Token algebra}
```

Comment:
i really, really dont agree with removing the definition of pointwise comparisons. If someone does not know what a partial order is they should not be guessing or looking it up